descriptor.md

```diff
@@ -176,6 +181,21 @@ In the following example, the descriptor indicates that the referenced manifest
 }
 ```
 
+In the following example, the descriptor indicates that the referenced manifest is retrievable from a particular URL:
+
+```json
+{
+  "mediaType": "application/vnd.example.signature+json",
+  "size": 3514,
+  "digest": "sha256:19387f68117dbe07daeef0d99e018f7bbf7a660158d24949ea47bc12a3e4ba17",
+  "reference": {
```

Comment:
What is the use case of adding a reference to a descriptor? I understand it in the context of of a manifest or index object, but adding this field to the descriptor is creating lots of strange potential implementations where I'm not seeing the value add in present use cases. Is this being proposed in expectation of a [descriptor as a manifest](https://github.com/opencontainers/distribution-spec/issues/252) being approved, or are there other use cases?